docs/_specification/1.2-DRAFT/data-entities.md

```diff
@@ -57,12 +57,12 @@ There is no requirement to represent _every_ file and folder in an RO-Crate as _
 - have metadata embedded in their filenames or paths which can be explained once,
```

Comment:
I do not understand this sentence. What does "which can be explained once" means in this context? @ptsefton maybe you can help clarify (according to Git the sentence was written by you).

Reply:
Added a fix